src/libraries/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.il

```diff
@@ -41,7 +41,7 @@
     01 00 00 00 00
   ) // false
   .hash algorithm 0x00008004
-  .ver 4:0:5:0
+  .ver 5:0:0:0
```

Comment:
Separate issue.  We should see about setting define's to flow the *actual* values from the build-system.  That way we get real fileversion and assembly version in ILProj-built assemblies.